frontend/src/app/render_wasm/api.cljs

```diff
@@ -469,6 +469,32 @@
         r4 (or (get corners 3) 0)]
     (h/call internal-module "_set_shape_corners" r1 r2 r3 r4)))
 
+
+(defn- translate-shadow-style
+  [style]
+  (case style
+    :drop-shadow 0
+    :inner-shadow 1
+    0))
+
+(defn set-shape-shadows
+  [shadows]
+
+  (h/call internal-module "_clear_shape_shadows")
+  (keep (fn [shadow]
+          (let [color (dm/get-prop shadow :color)
+                blur (dm/get-prop shadow :blur)
+                rgba (rgba-from-hex (dm/get-prop color :color) (dm/get-prop color :opacity))
+                hidden (dm/get-prop shadow :hidden)
+                x (dm/get-prop shadow :offset-x)
+                y (dm/get-prop shadow :offset-y)
+                spread (dm/get-prop shadow :spread)
```

Comment:
spread is set but not used when rendering the shadow, this actually generates different shadows than when using svg render